app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -324,6 +326,13 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
                 view = view
             )
 
+            pipFeature = PictureInPictureFeature(
```

Comment:
Did we want to consider a nightly-only feature flag?

Reply:
good call at least for a bit